e2e/repeater.e2e-spec.ts

```diff
@@ -4,15 +4,6 @@ import {
 } from '@skyux-sdk/e2e';
 
 describe('Repeater', () => {
-  it('should match the baseline sort screenshot', (done) => {
-    SkyHostBrowser.get('visual/sort');
```

Comment:
I have no idea why a test for `sort` was in here ☝️ .

Reply:
Funny!